src/projectController.ts

```diff
@@ -197,7 +197,8 @@ export class ProjectController implements vscode.Disposable {
         this.folderToProjectsMap.clear();
         if (vscode.workspace.workspaceFolders) {
             for (const folder of vscode.workspace.workspaceFolders) {
-                await this.addFolder(folder);
+                const projects: CMakeProject[] = await this.addFolder(folder);
+                this.afterAddFolderEmitter.fire({ folder: folder, projects:  projects});
```

Comment:
Should this always be fired after `this.addFolder`?  If so, then `this.addFolder` should do it so that we never make this mistake again.

Reply:
I agree